packages/snfoundry/scripts-ts/deploy-contract.ts

```diff
@@ -4,14 +4,25 @@ import { networks } from "./helpers/networks";
 import yargs from "yargs";
 import { CallData, hash } from "starknet-dev";
 import { Network } from "./types";
-import { LegacyContractClass, CompiledSierra, RawArgs } from "starknet";
+import { LegacyContractClass, CompiledSierra, RawArgs, getChecksumAddress } from "starknet";
 
 const argv = yargs(process.argv.slice(2)).argv;
 const networkName: string = argv["network"];
 
 let deployments = {};
 
 const { provider, deployer }: Network = networks[networkName];
+
+const checkWalletDeployed = async (walletAddress: string): Promise<boolean> => {
+  try {
+    const accountInfo = getChecksumAddress(walletAddress);
```

Comment:
the funciton `getCheckSumAdddress` is to validate and format any address, however it doesnt tell me whether this address is deployed or not.